python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -724,6 +724,39 @@ def plot_solution_type(self):
 
         self._add_figure(name="solution_type", figure=figure, title="Solution Type")
 
+    def plot_stationary_status(self):
+        """!
+        @brief Plot the staionary status over time.
+        """
+        if self.output_dir is None:
+            return
+
+        # Read the pose data.
+        result = self.reader.read(message_types=[PoseMessage], source_ids=self.default_source_id, **self.params)
+        pose_data = result[PoseMessage.MESSAGE_TYPE]
+
+        if len(pose_data.p1_time) == 0:
+            self.logger.info('No pose data available. Skipping solution type plot.')
+            return
+
+        # Set up the figure.
+        figure = make_subplots(rows=1, cols=1, print_grid=False, shared_xaxes=True, subplot_titles=['Stationary Status'])
+
+        figure['layout']['xaxis'].update(title=self.p1_time_label)
+        figure['layout']['yaxis1'].update(title="Stationary Status",
+                                          ticktext=['Nonstationary', 'Stationary'],
+                                          tickvals=[0, 1])
```

Comment:
You should use `PoseMessage.FLAG_STATIONARY` instead of 1 here since below you're just doing a &, not converting it to a bool or something. Technically if this bit value was anything other than 0x1, this wouldn't work correctly.